eth2util/network.go

```diff
@@ -0,0 +1,90 @@
+// Copyright © 2022 Obol Labs Inc.
+//
+// This program is free software: you can redistribute it and/or modify it
+// under the terms of the GNU General Public License as published by the Free
+// Software Foundation, either version 3 of the License, or (at your option)
+// any later version.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of  MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+// more details.
+//
+// You should have received a copy of the GNU General Public License along with
+// this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package eth2util
+
+import (
+	"fmt"
+
+	"github.com/obolnetwork/charon/app/errors"
+)
+
+// Network contains information about an Ethereum network.
+type Network struct {
+	// ChainID represents chainID of the network.
+	ChainID int64
+	// Name represents name of the network.
+	Name string
+	// ForkVersion represents fork version of the network in hex.
+	ForkVersion string
+}
+
+var (
+	Mainnet = Network{
```

Comment:
maybe prefix names with `Network`?

Reply:
Not sure, will make it more verbose i guess ?